src/subscription_manager/gui/registergui.py

```diff
@@ -277,17 +324,16 @@ def __init__(self, backend, facts, reg_info=None,
         # To update the 'next/register' button in the parent dialog based on the new page
         self.register_notebook.connect('switch-page',
                                        self._on_switch_page)
-
-        screen_classes = [ChooseServerScreen, ActivationKeyScreen,
-                          CredentialsScreen, OrganizationScreen,
-                          EnvironmentScreen, PerformRegisterScreen,
-                          SelectSLAScreen, ConfirmSubscriptionsScreen,
-                          PerformSubscribeScreen, RefreshSubscriptionsScreen,
-                          InfoScreen, DoneScreen]
+        self.screen_classes = screen_classes or [ChooseServerScreen, ActivationKeyScreen,
+                                                 CredentialsScreen, OrganizationScreen,
+                                                 EnvironmentScreen, PerformRegisterScreen,
+                                                 SelectSLAScreen, ConfirmSubscriptionsScreen,
+                                                 PerformSubscribeScreen, RefreshSubscriptionsScreen,
+                                                 InfoScreen, DoneScreen]
         self._screens = []
 
         # TODO: current_screen as a gobject property
-        for idx, screen_class in enumerate(screen_classes):
+        for idx, screen_class in enumerate(self.screen_classes):
```

Comment:
At some point in the future (read, not this branch/release) the 'screens' info and logic likely needs to move to it's own class or interface (ie, register_gui.RegisterScreens that knows the screens and default ordering, and stuff like set_current_screen and the screen enums, etc)